src/Internal/Types/Interval.purs

```diff
@@ -1,32 +1,20 @@
 module Ctl.Internal.Types.Interval
-  ( AbsTime(AbsTime)
+  ( AbsTime(..)
   , Closure
-  , Extended(NegInf, Finite, PosInf)
-  , Interval(EmptyInterval, StartAt, EndAt, AlwaysInterval, FiniteInterval)
-  , LowerBound(LowerBound)
-  , ModTime(ModTime)
-  , OnchainPOSIXTimeRange(OnchainPOSIXTimeRange)
-  , POSIXTime(POSIXTime)
+  , Extended(..)
+  , Interval(..)
+  , LowerBound(..)
+  , ModTime(..)
+  , OnchainPOSIXTimeRange(..)
+  , POSIXTime(..)
   , POSIXTimeRange
-  , PosixTimeToSlotError
-      ( CannotFindTimeInEraSummaries
-      , PosixTimeBeforeSystemStart
-      , StartTimeGreaterThanTime
-      , EndSlotLessThanSlotOrModNonZero
-      , CannotGetBigIntFromNumber'
-      , CannotGetBigNumFromBigInt'
-      )
-  , RelSlot(RelSlot)
-  , RelTime(RelTime)
+  , PosixTimeToSlotError(..)
+  , RelSlot(..)
+  , RelTime(..)
```

Comment:
Please undo these changes. We use explicit imports and exports.

Reply:
Done.